ui.tests/test-module/specs/actions/viewQualifiedName/viewQualifiedName.authoring.spec.js

```diff
@@ -0,0 +1,238 @@
+const afConstants = require("../../../libs/commons/formsConstants");
+const sitesSelectors = require("../../../libs/commons/sitesSelectors");
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+describe("View Qualified Name Tests", () => {
```

Comment:
All these test run should be under cy.isLatestAddon check, you can check the existing references, thats why the test cases are failing with released addon.